### PR TITLE
Add external link component

### PR DESCRIPTION
**Pull request by 0xjocke**
image

### CODE REVIEW

**Review by noisekit — APPROVED** [review]

**Comment by bachstatter** [comment]
Oh I'm messing up a bit here. Wasn't suppose to inlcude the WIP transaction modal....

File: v2/components/icons/ArrowTopRight/ArrowTopRight.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+export const ArrowTopRight = ({
+  width = 10,
+  height = 12,
+  color = '#00D1FF',
+  ...props
+}: ArrowTopRightProps) => {
+  return (
+    <Icon width={`${width}px`} height={`${height}px`} viewBox="0 0 8 9" {...props}>
```
Comment (jmzwar) [inline v2/components/icons/ArrowTopRight/ArrowTopRight.tsx:17]: Don't need to do here - but we can actually remove this string literal logic. Icon takes either string or number

Comment (jmzwar) [reply v2/components/icons/ArrowTopRight/ArrowTopRight.tsx:17]: I may need to update some other icons

**Review by jmzwar — APPROVED** [review]

File: v2/components/icons/ArrowTopRight/ArrowTopRight.tsx (hunk 1 above)
Comment (bachstatter) [reply v2/components/icons/ArrowTopRight/ArrowTopRight.tsx:17]: Hmm yea sorry lazy copy paste that's spreading.. I'm struggling a bit with another icon right now

Comment (bachstatter) [reply v2/components/icons/ArrowTopRight/ArrowTopRight.tsx:17]: hmm without the string it's using the sizing vars rather than pixel though

**Pull request merged**